### DISCUSSION TITLE
More Oscilators

### DISCUSSION THREAD

**Discussion by tripsylon** [bangcorrupt/hacktribe#256] [Q&A] [answered]
Will there be a way to add more oscilators from the synth version, or manipulate the settings on the existing ones, wia midi or on a computer. Thank You.

**Comment by bangcorrupt** [comment, marked as answer]
We already have all of the synth oscillators, the missing sounds are sample instruments.  Hopefully at some point I will get multi-sampling working, then all of the synth PCM instruments should load correctly.  Editing parameters for the existing oscillators will be an option in Hacktribe Editor at some point, maybe also a shift parameter for the Osc Edit knob.

**Reply by enorrmann** [reply]
multi-sampling woud be huge !

**Reply by tripsylon** [reply]
I see, the only oscillator I couldn’t find was the boost saw, Is it somewhere between the x saw under another name?  And could you explain what is the concept behind the different x saw osc’s , how  do they function, how are they all tuned etc ? Thank you

**Reply by bangcorrupt** [reply]
Boost Saw is a single cycle PCM sample, the Edit knob applies a drive effect similar to the other samples in the synth firmware.  The Boost Saw waveform is in the factory sample set for the sampler, hopefully I can port the drive effect some day.  

The 'X' oscillators are single operator frequency modulation, the numbers in the name are the ratio of the modulator to the carrier.
